modules/mono/editor/GodotTools/GodotTools/Build/BuildSystem.cs

```diff
@@ -50,7 +50,7 @@ public static class BuildSystem
                 process.OutputDataReceived += (_, e) => stdOutHandler.Invoke(e.Data);
             if (stdErrHandler != null)
                 process.ErrorDataReceived += (_, e) => stdErrHandler.Invoke(e.Data);
-
+            process.StartInfo.StandardOutputEncoding = Encoding.UTF8;
```

Comment:
This would be better defined together with other `startInfo` properties before it's passed to the `Process` constructor:
```
            startInfo.RedirectStandardOutput = true;
            startInfo.RedirectStandardError = true;
            startInfo.UseShellExecute = false;
            startInfo.CreateNoWindow = true;
            startInfo.StandardOutputEncoding = Encoding.UTF8;
            startInfo.EnvironmentVariables["DOTNET_CLI_UI_LANGUAGE"]
                = ((string)editorSettings.GetSetting("interface/editor/editor_language")).Replace('_', '-');
```

Same for the other one below.